live-event-app/live-event-app/RootViewController.swift

```diff
@@ -0,0 +1,159 @@
+        
+    messageListContainerViewController.view.translatesAutoresizingMaskIntoConstraints = false
+    streamViewController.view.translatesAutoresizingMaskIntoConstraints = false
+    
+    // Creates constraints for Portrait and Landscape orientations
+    portraitConstraints = [
+      messageListContainerViewController.view.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+      messageListContainerViewController.view.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+      messageListContainerViewController.view.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+      messageListContainerViewController.view.heightAnchor.constraint(equalTo: view.heightAnchor, multiplier: 9.0 / 16.0),
+      streamViewController.view.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+      streamViewController.view.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+      streamViewController.view.topAnchor.constraint(equalTo: view.topAnchor),
+      streamViewController.view.bottomAnchor.constraint(equalTo: messageListContainerViewController.view.topAnchor)
+    ]
+    
+    landscapeConstraints = [
+      streamViewController.view.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+      streamViewController.view.trailingAnchor.constraint(equalTo: messageListContainerViewController.view.leadingAnchor),
+      streamViewController.view.topAnchor.constraint(equalTo: view.topAnchor),
+      streamViewController.view.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+      
+      messageListContainerViewController.view.leadingAnchor.constraint(equalTo: streamViewController.view.trailingAnchor),
+      messageListContainerViewController.view.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+      messageListContainerViewController.view.widthAnchor.constraint(equalTo: view.widthAnchor, multiplier: 7.0 / 16.0),
+      messageListContainerViewController.view.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+      messageListContainerViewController.view.topAnchor.constraint(equalTo: view.topAnchor)
+    ]
+    
+    // Creates a debug overlay to display the FPS meter
+    let metricsView = MetricsView()
+    view.addSubview(metricsView)
+    view.bringSubviewToFront(metricsView)
+    
+    try? metricsView.activate {[
+      $0.topAnchor.constraint(equalTo: $1.topAnchor, constant: 40),
+      $0.trailingAnchor.constraint(equalTo: $1.trailingAnchor, constant: -20)
+    ]}
+        
+    // Activates a set of constraints depending on the orientation
+    NSLayoutConstraint.activate(
+      scene.interfaceOrientation.isPortrait ? portraitConstraints : landscapeConstraints
+    )
+  }
+  
+  private func getMessageListCustomTitleView() -> UIView {
+    
+    let customTitleView = UILabel()
+    customTitleView.font = UIFont(name: "Poppins-Bold", size: 14)
+    customTitleView.textColor = UIColor(named: "MessageList.NavigationBar.TitleTextColor")
+    customTitleView.text = "STREAM CHAT"
+    customTitleView.sizeToFit()
```

Comment:
I would move this customTitleView to separate class, but it is not so important

Reply:
`customTitleView` is in fact a `UILabel` object. I created it because there's no possibility to theme the navigation title string with custom font, foreground color, etc.